gunicorn/reloader.py

```diff
@@ -12,10 +12,11 @@
 import threading
 
 COMPILED_EXT_RE = re.compile(r'py[co]$')
+POLL_INTERVAL = float(os.getenv('GUNICORN_RELOADER_POLL_INTERVAL', '1'))
```

Comment:
Should we expose this on the CLI somehow? Or is this intended just for the tests?

Reply:
This is just intended for the tests, so the test does not take multiple seconds each time.